polaris-react/src/components/Scrollable/Scrollable.scss

```
@@ -21,6 +23,62 @@
       var(--p-color-border-interactive-focus);
     outline-offset: var(--p-space-05);
   }
+
+  &.hasTopShadow,
+  &.hasBottomShadow {
+    /* Why not combine the box shadows onto one element? Because then it would
+     * need to be height: 100%, but that doesn't work when there's no explicit
+     * height set on the parent element (eg; for the Modal component which uses
+     * min-height: 80vh instead). */
```

Let's let the comment wrap naturally based on the size of the editor page

```suggestion
    /* Why not combine the box shadows onto one element? Because then it would need to be height: 100%, but that doesn't work when there's no explicit height set on the parent element (eg; for the Modal component which uses min-height: 80vh instead). */
```

---

Prettier keeps reformatting this to be multiline... Not sure if it's our project config or my editor (neovim, which should just be using the project config) 🤔